addons/test_mail/models/test_mail_models.py

```diff
@@ -143,7 +143,18 @@ class MailTestStandard(models.Model):
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True)
     container_id = fields.Many2one('mail.test.container', tracking=True)
     company_id = fields.Many2one('res.company')
-
+    track_hidden_fields = fields.Char()  # comma-separated list
+
+    def _track_filter_for_display(self, tracking_values):
+        values = super()._track_filter_for_display(tracking_values)
+        hidden_fields = set(self.track_hidden_fields.split(',') if self.track_hidden_fields else '')
+        return values.filtered(lambda val: val.field_id.name not in hidden_fields)
+
+    def _track_get_default_log_message(self, changes):
+        hidden_fields = set(self.track_hidden_fields.split(',') if self.track_hidden_fields else '')
+        if not all(change in hidden_fields for change in changes):
+            return f'There was a change on {self.name} for fields "{",".join(changes)}"'
```

Comment:
To avoid changes in test_mail_message.py, maybe
```suggestion
        if filtered_fields:
            return f'Filtered fields {",".join(changes)} have been updated on {self.name}, triggering custom log."
```

Reply:
I feel like that's too specific to the one test. I'll just add a field to enable it, not like it costs anything, and I guess it's more test-oriented too :)